lsudt/lsudt.py

```diff
@@ -474,10 +496,86 @@ def init_argparse() -> argparse.ArgumentParser:
         dest="id_path",
         help="limit output to devices with a parent starting with given idpath",
     )
+    parser.add_argument(
+        "--extract-env",
+        "-x",
+        action="store_true",
+        dest="extract_env",
+        help="print name=value pairs to stdout"
+    )
 
     return parser
 
 
+def generate_and_print_env_strings(label: str):
+    """
+    prints the name=value pairs to stdout
+    """
+
+    strings = ""
+    for key in list(env_pairs.keys()):
+        counter = 0
+        for dev in env_pairs[key]:
+            strings += (f" {label}_{key}{counter}={dev} ")
+            counter+=1
+    print(strings)
+
+
+def handle_usb_children(usb_device, args):
+    """
+    similiar to showtree
+    """
+
+    if len(usb_device.children) != 0:
+        for child_dev in usb_device.children:
+            env_pair_strings(child_dev, args)
+
+
+def fix_label(label : str):
```

Comment:
Please add a description. And rename to indicate what label this fixes, e.g. fix_env_label or similar